spec/swagger_spec.rb

```diff
@@ -11,8 +11,42 @@
 
   let(:swagger_routes) { Api::Docs.routes }
 
-  it "routes match" do
-    expect(rails_routes).to match_array(swagger_routes)
+  describe "Routing" do
+    include Rails.application.routes.url_helpers
+
+    it "routes match" do
+      expect(rails_routes).to match_array(swagger_routes)
+    end
+
+    it "allows customizable routes" do
```

Comment:
Can you also test for the default value?

Reply:
All of the other tests test that.  If you want I can add a single expectation like we have in this test for the default case

Reply:
No this is fine then. I was confused because `api` was being removed from two tests in this PR.

Reply:
Ah.  Yeah, the test above compares the rails routes to the swagger routes and the swagger routes all start with "/api/v0.0" since that's the base path in the swagger file.